app/views/orders/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_for [order] do |f| %>
+    <div class="field">
+        <% if @customer.present? %>
+            <% customers_full_name_array = Customer.all.map { |customer| [customer.full_name, customer.id] } %>
```

Comment:
In general, you'll want to take as much logic out of the view as possible. This is an array you can compute and send from the controller if you want to. 

You kind of already did this in sending `@customers` over. So you can replace `Customer.all` here with `@customers`.